Before this patch, with CPP exported:
```
checking for inet_aton function prototype in arpa/inet.h... ./configure: 6386: CPP: not found
checking how to run the C preprocessor... x86_64-pc-linux-gnu-gcc -E
./configure: 6512: ac_fn_c_try_cpp: not found
./configure: 6512: ac_fn_c_try_cpp: not found
configure: error: in `/var/tmp/portage/net-analyzer/dhcp_probe-1.3.1/work/dhcp_probe-1.3.1':
configure: error: C preprocessor "x86_64-pc-linux-gnu-gcc -E" fails sanity check
See `config.log' for more details
```

... and unexported:
```
checking for inet_aton function prototype in arpa/inet.h... ./configure: 6386: CPP: not found
[... continues without dying ...]
```
--- a/configure.ac
+++ b/configure.ac
@@ -16,6 +16,7 @@ dnl Checks for programs.
 AC_PROG_CC
 AC_PROG_RANLIB
 AM_PROG_AR
+AC_PROG_CPP
 
 dnl ##########################################################################
 
